tests/testthat/test-custom_disconnect_message.R

```diff
@@ -0,0 +1,118 @@
+test_that("publication link is valid", {
+  # Test that an EES publication link passes
+  expect_no_error(
+    custom_disconnect_message(
+      refresh = "Refresh page",
+      links = "https://department-for-education.shinyapps.io/dfe-shiny-template/?_inputs_&navlistPanel=%22dashboard%22&tabsetpanels=%22Valuebox%20example%22&cookieAccept=1&cookieReject=0&cookieLink=0&hideAccept=1&hideReject=0&remove=0&selectPhase=%22All%20LA%20maintained%20schools%22&selectArea=%22England%22&selectBenchLAs=null",
+      publication_name = "Pupil attendance in schools",
+      publication_link = "https://explore-education-statistics.service.gov.uk/find-statistics/pupil-attendance-in-schools"
+    )
+  )
+
+  # Test that a gov.uk publication link passes
+  expect_no_error(
+    custom_disconnect_message(
+      refresh = "Refresh page",
+      links = "https://department-for-education.shinyapps.io/dfe-shiny-template/?_inputs_&navlistPanel=%22dashboard%22&tabsetpanels=%22Valuebox%20example%22&cookieAccept=1&cookieReject=0&cookieLink=0&hideAccept=1&hideReject=0&remove=0&selectPhase=%22All%20LA%20maintained%20schools%22&selectArea=%22England%22&selectBenchLAs=null",
```

Comment:
Do we need to use a URL this long, could it just be "https://department-for-education.shinyapps.io/dfe-shiny-template/"?